snaps/how-to/test-a-snap.md

```diff
@@ -0,0 +1,88 @@
+---
+description: Use Jest for end-to-end Snap testing.
+sidebar_position: 5
+---
+
+# Test a Snap
+
+Use the [`@metamask/snaps-jest`](https://github.com/MetaMask/snaps/tree/main/packages/snaps-jest)
+package for end-to-end Snap testing in a Jest environment.
+
+## Steps
+
+### 1. Install @metamask/snaps-jest
+
+Install the `@metamask/snaps-jest` package into your Snap project using [Yarn](https://yarnpkg.com/)
+or [npm](https://www.npmjs.com/):
+
+```bash
+yarn add -D @metamask/snaps-jest
+```
+
+or
+
+```bash
+npm i @metamask/snaps-jest
+```
+
+### 2. Configure @metamask/snaps-jest
+
+The easiest way to configure this package is to add it to your Jest configuration as a preset.
+In the `jest.config.js` file, add the following:
+
+```js title="jest.config.js"
+module.exports = {
+  preset: '@metamask/snaps-jest',
+};
+```
+
+This automatically configures Jest to use the `@metamask/snaps-jest` environment, and to use the
+`@metamask/snaps-jest` matchers.
+You can then run the `jest` command as usual.
+
+:::note
+`@metamask/snaps-jest` assumes the Snap is built in the directory you run Jest from.
+If you use a different directory, you can specify the path using the
+[`root`](../reference/cli/options.md#r-root) option, or by running your own HTTP server.
+It currently is not possible to use `@metamask/snaps-jest` with a Snap that is not built.
```

Comment:
Maybe this is better... It's not currently possible to... 

Or... Currently, it's not possible to...